### ISSUE TITLE
Send Results scheduled job fails

### ISSUE THREAD

**Issue by RaduCStefanescu** [code4romania/stam-acasa#343]
The Send Results scheduled job fails with the following error:
```
2020-04-23 14:50:00 [EROR]<29> [StamAcasa.JobScheduler.Jobs.SendResultsNotificationJob] Error when dispatching results notification: Object reference not set to an instance of an object..
System.NullReferenceException: Object reference not set to an instance of an object.
   at StamAcasa.Common.Notifications.ResultNotificationsDispatch.CreateCountyEmailData(String county) in /src/StamAcasa.Common/Notifications/ResultNotificationsDispatch.cs:line 110
   at StamAcasa.Common.Notifications.ResultNotificationsDispatch.Process() in /src/StamAcasa.Common/Notifications/ResultNotificationsDispatch.cs:line 52
   at StamAcasa.JobScheduler.Jobs.SendResultsNotificationJob.RunAsync(CancellationToken cancellationToken) in /src/StamAcasa.JobScheduler/Jobs/SendResultsNotificationJob.cs:line 27
```

**Comment by idormenco** [comment]
i think this was fixed in this pr #345

**Comment by RaduCStefanescu** [comment]
I'll try to keep this one open until I can see the Results dispatch fully working.

**Comment by RaduCStefanescu** [comment]
This issue was a combinaton of misconfiguration and bad error management, which was fixed in #345 
I am going to close it - thank you @idormenco and @thewindev
